bootstrap/kernel_bootstrap.php

```diff
@@ -25,9 +25,18 @@
 }
 
 $xml = new \SimpleXMLElement(file_get_contents($phpUnitFile));
-$kernelDir = $xml->php[0]->server[0]['value'];
 
-$kernelFile = $rootDir.'/'.$kernelDir.'/AppKernel.php';
+$envDir = $xml->xpath("//php/server[@name='KERNEL_DIR']");
+if (!count($envDir)) {
```

Comment:
so this means that kernel dir MUST be set? i would say we should allow it to not be set and only have KERNEL_CLASS in the phpunit.xml file. now we require both.

Reply:
`KERNEL_DIR` was required before my PR, if there was no `<server name="KERNEL_DIR" .../>` in phpunit.xml.dist, that there was error. I just added exception